packages/formbricks-api-wrapper/src/api/client/index.ts

```diff
@@ -0,0 +1,15 @@
+import { ResponseAPI } from "./response";
+import { DisplayAPI } from "./display";
+import { ApiConfig } from "../../types";
+
+export class Client {
+  response: ResponseAPI;
```

Comment:
I think, having singular is better since it's a method name and can differ from the API route! Singular keeps it simple and does not set any expectations